packages/js-dapi-client/lib/BlockHeadersProvider/BlockHeadersProvider.js

```diff
@@ -35,15 +49,14 @@ class BlockHeadersProvider extends EventEmitter {
       ...options,
     };
 
-    this.spvChain = new SpvChain(this.options.network);
-    this.started = false;
-  }
+    this.spvChain = this.options.spvChain || new SpvChain(this.options.network, 100);
```

Comment:
Shall we move required deps from opts to the constuctor params?

Reply:
Fixed